src/page.rs

```diff
@@ -261,8 +261,10 @@ impl Page {
         Ok(resp.result.node)
     }
 
-    pub async fn close(self) {
-        todo!()
+    /// Closes page
+    pub async fn close(self) -> Result<()> {
+        self.execute(CloseParams {}).await?;
```

Comment:
Can you add a short note that indicates that this `Tries to close page, running its `beforeunload` hooks, if any` as well as a reference to `CloseParams`